lib/diagnostic.rb

```diff
@@ -91,13 +91,16 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+# The rake test failed for below, but this way still works.
+Diagnostic.remove_from_array = 'arr.pop(2)'
 
```

Comment:
the test looks for `arr.pop 2` but you're dead right, we noticed the test issues yesterday and haven't pushed the changes yet 👌